### PR TITLE
804 update config

### PR DESCRIPTION
**Pull request by ArashDai**

### CODE REVIEW

File: src/app/manager-dashboard/manager-dashboard.component.ts [hunk 1]
```diff
@@ -10,6 +10,7 @@ import { UserService } from '../shared/user.service';
           i18n mat-raised-button>{{ planetType === 'center' ? 'Nation' : 'Community' }}</a>
       </span>
       <a routerLink="/feedback" i18n mat-raised-button>Feedback</a>
+      <a routerLink="configuration/update/{{configurationId}}" il8n mat-raised-button>Configuration</a>
```
Comment (lmmrssa) [inline src/app/manager-dashboard/manager-dashboard.component.ts]: There will be only one configuration per planet so configuration id will not be required to pass

File: src/app/manager-dashboard/manager-dashboard.component.ts [hunk 2]
```diff
@@ -26,6 +27,7 @@ export class ManagerDashboardComponent implements OnInit {
   displayDashboard = true;
   message = '';
   planetType = this.userService.getConfig().planetType;
+  configurationId = this.userService.getConfig()._id
```
Comment (lmmrssa) [inline src/app/manager-dashboard/manager-dashboard.component.ts]: this code is not required

File: .vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,2 @@
+{
```
Comment (lmmrssa) [inline .vscode/settings.json]: unnecessary file

**Comment by ArashDai** [comment]
not sure what happened with the last commit, trying to fix it now

**Comment by lmmrssa** [comment]
@ArashDai done some fixing for codeclimate and linting. But there are still errors so could you go through it again. As it is marked WIP hope you still have changes to make.

**Comment by ArashDai** [comment]
I'll fix the style issues, my linter isn't working for some reason

**Comment by lmmrssa** [comment]
closing this PR

**Pull request closed without merging**